roles/openshift_aws/tasks/vpc.yml

```diff
@@ -1,4 +1,19 @@
 ---
+- name: query azs
+  aws_az_facts:
+  register: azs
+
+- debug:
+    msg:
```

Comment:
This message should be part of failed_when, not it's own debug statement.

Reply:
Ah!  TIL about "fail: msg="!